react-js/react-part-2.md

```diff
@@ -0,0 +1,184 @@
+# React Part 2
+
+## Projected Time
+
+1.5 hours
+- Lesson: 30min
+- Guided Practice: 30min
```

Comment:
Please increase these values as well. There's no way a new person would get through this in 1.5 hours. Have someone try it out to get a time estimate or make your best guess, erring on the side of it being an overestimate.